README-hi.md

```diff
@@ -0,0 +1,82 @@
+
+स्वागत हे! इस रिपॉजिटरी में [कुबरनेट्स वेबसाइट और दस्तावेज](https://kubernetes.io/) बनाने के लिए आवश्यक सभी संपत्तियां हैं। हम बहुत खुश हैं कि आप योगदान करना चाहते हैं!
+
+## डॉक्स में योगदान देना
+
+आप अपने GitHub खाते में इस रिपॉजिटरी की एक copy बनाने के लिए स्क्रीन के ऊपरी-दाएँ क्षेत्र में **Fork** बटन पर क्लिक करें। इस copy को *Fork* कहा जाता है। अपने fork में कोई भी परिवर्तन करना चाहते हैं, और जब आप उन परिवर्तनों को हमारे पास भेजने के लिए तैयार हों, तो अपने fork पर जाएं और हमें इसके बारे में बताने के लिए एक नया pull request बनाएं।
+
+एक बार जब आपका pull request बन जाता है, तो एक कुबरनेट्स समीक्षक स्पष्ट, कार्रवाई योग्य प्रतिक्रिया प्रदान करने की जिम्मेदारी लेगा। pull request के मालिक के रूप में, **यह आपकी जिम्मेदारी है कि आप कुबरनेट्स समीक्षक द्वारा प्रदान की गई प्रतिक्रिया को संबोधित करने के लिए अपने pull request को संशोधित करें।**
+
+यह भी ध्यान दें कि आप एक से अधिक कुबरनेट्स समीक्षक आपको प्रतिक्रिया प्रदान कर सकते हैं या आप एक कुबरनेट्स समीक्षक से प्रतिक्रिया प्राप्त कर सकते हैं जो मूल रूप से आपको प्रतिक्रिया प्रदान करने के लिए दिए गए एक से भिन्न है। इसके अलावा, कुछ मामलों में, आपका एक समीक्षक जरूरत पड़ने पर [कुबेरनेट्स टेक समीक्षक](https://github.com/kubernetes/website/wiki/Tech-reviewers) से तकनीकी समीक्षा प्राप्त कर सकता है। समीक्षक समय पर प्रतिक्रिया देने के लिए पूरी कोशिश करेंगे, लेकिन परिस्थितियों के आधार पर प्रतिक्रिया समय अलग-अलग हो सकता है।
+
+कुबरनेट्स प्रलेखन में योगदान देने के बारे में अधिक जानकारी के लिए, देखें:
+
+* [योगदान देना शुरू करें](https://kubernetes.io/docs/contribute/start/)
+* [Staging Your Changes](http://kubernetes.io/docs/contribute/intermediate#view-your-changes-locally)
+* [पेज टेम्पलेट](http://kubernetes.io/docs/contribute/style/page-templates/)
+* [प्रलेखन शैली गाइड](http://kubernetes.io/docs/contribute/style/style-guide/)
+* [स्थानीयकरण कुबरनेट्स प्रलेखन](https://kubernetes.io/docs/contribute/localization/)
+
+## `README.md`'s स्थानीयकरण कुबरनेट्स प्रलेखन
+
+आप पर हिंदी स्थानीयकरण के maintainers तक पहुँच सकते हैं:
+
+* Yashu Mittal ([Twitter](https://twitter.com/mittalyashu77), [GitHub](https://github.com/mittalyashu))
+* [Slack channel](https://kubernetes.slack.com/messages/kubernetes-docs-hi)
+
+## स्थानीय रूप से Docker का उपयोग करके साइट चलाना
+
+कुबरनेट्स वेबसाइट को स्थानीय रूप से चलाने के लिए अनुशंसित तरीका एक विशेष [Docker](https://docker.com) image को चलाना है, जिसमें [Hugo](https://gohugo.io) स्टेटिक साइट जनरेटर शामिल है।
+
+> यदि आप विंडोज पर हैं, तो आपको कुछ और टूल्स की आवश्यकता होगी जिन्हें आप [Chocolatey](https://chocolatey.org) के साथ इंस्टॉल कर सकते हैं।
+
+> यदि आप Docker के बिना स्थानीय रूप से वेबसाइट चलाना पसंद करते हैं, तो नीचे Hugo का उपयोग करके स्थानीय रूप से साइट चलाना देखें।
+
+यदि आप Docker के बिना स्थानीय रूप से वेबसाइट चलाना पसंद करते हैं, तो नीचे दिए गए Hugo का उपयोग करके स्थानीय रूप से [साइट को चलाने](#running-the-site-locally-using-hugo) का तरीका देखें।
+
+यदि आप [डोकर](https://www.docker.com/get-started) चला रहे हैं, तो स्थानीय रूप से `कुबेरनेट्स-ह्यूगो` Docker image बनाएँ:
```

Comment:
Should we write it as डॉकर or डोकर? https://dictionary.cambridge.org/dictionary/english/docker US pronunciation sounds like  डॉकर but the I'm not able to decide if UK pronunciation sounds like डॉकर or  डोकर

Reply:
Now, this one is a bit tricky 😅

Reply:
I did some digging and found that

### डोकर

The pronunciation of the word **डोकर** is `do-kar`

### डॉकर

The pronunciation of the word **डॉकर** is `doc-kr`

I know the pronunciation of both the word is almost the same but `डॉकर` is the correct word.

Reply:
Lets go with डॉकर!

Reply:
Now we have found the official `डॉकर` word, should we change all Docker word to `डॉकर`?